services/workflows-service/src/supabase/supabase.module.ts

```diff
@@ -1,8 +1,9 @@
 import { Module } from '@nestjs/common';
 import { SupabaseService } from './supabase.service';
+import { SentryService } from '../sentry/sentry.service';
 
 @Module({
-  providers: [SupabaseService],
-  exports: [SupabaseService],
```

Comment:
Please remove SentryService from providers+ exports and use
```
import { SentryModule } from '@/sentry/sentry.module';


imports: [SentryModule],
```

than the service will be injectable in your service